networking/synthetic/stac-prepare-nodes.yaml

```diff
@@ -2,7 +2,48 @@
 - hosts: stac_nodes
   remote_user: root
   tasks:
-    
+
+  - name: install epel 7
+    yum:
+      name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      state: present
+
+  - name: add rhel latest repos
+    yum_repository:
+      name: rhel_latest
+      description: RHEL perf-dept repo
+      baseurl: http://download.devel.redhat.com/rel-eng/latest-RHEL-7/compose/Server/x86_64/os/
+
+  - name: install reqiured pkgs
+    yum: state=present name={{ item }}
+      with_items:
+      - kernel
+      - kernel-devel
+      - gcc
+      - js
+      - tuna
+      - tuned
+      - numactl
+      - python34
+
+  - name: download openonload
+    get_url:
+      url: http://www.openonload.org/download/openonload-201606-u1.3.tgz
+      dest: /root/
+
+  - name: unarchive openonload
+    unarchive:
+        src: /root/openonload-201606-u1.3.tgz
+        dest: /root/
+
+  - name: install openonload
+    script: /root/openonload-201606-u1.3/scripts/onload_install
```

Comment:
Not a big deal, just not sure why you don't use RPMs for this.

You can do

```rpmbuild -ta openonload-201606-u1.3.tgz``` then you will get RPM of openonload to install.

Reply:
done